Submitted By:            Randy McMurchy <randy_at_linuxfromscratch_dot_org>
Date:                    2005-10-02
Initial Package Version: 0.4.2
Upstream Status:         Unknown
Origin:                  Redhat Development CVS
            http://cvs.fedora.redhat.com/viewcvs/devel/ots/ots-0.4.2-gcc4.patch?rev=1.1&view=markup
Description:             Fixes a GCC-4.0.x build problem


--- ots-0.4.2/src/article.c.orig	2005-03-02 19:42:19.479330826 +0000
+++ ots-0.4.2/src/article.c	2005-03-02 19:42:43.967625287 +0000
@@ -35,7 +35,7 @@
 ots_new_sentence (void)
 {
   OtsSentence *aLine = g_new0 (OtsSentence, 1);
-  (GList *) aLine->words = NULL;
+  aLine->words = NULL;
   aLine->wc = 0;
   aLine->selected = 0;
   aLine->score = 0;

